src/app/dashboard/Missions/DashboardMissions.jsx

```diff
@@ -133,15 +133,24 @@ const PageButtons = ({ classes, pageView, setPageView }) => {
 
 const DashboardMissions = ({ inDone, inPlanning, inProgress, inProposed }) => {
   const classes = useStyles();
-  const all = { inProposed, inPlanning, inProgress, inDone };
 
   const [pageView, setPageView] = useState(pageViews.list);
-  let filtered = inProposed;
+  const [selectedMission, setSelectedMission] = useState(null);
 
+  const all = { inProposed, inPlanning, inProgress, inDone };
```

Comment:
Would it be more clear to pass this object in named as missions from the parent, and access or destructure this similar to  props.missions.inProgress ?